### DISCUSSION TITLE
Removing billing fields from checkout

### DISCUSSION THREAD

**Discussion by virajsoni06** [swellstores/community#248] [Help] [answered]
Hi, we have a custom front-end and are using Swell's inbuilt checkout for our checkouts. 
We want to remove a number of billing fields which aren't relevant to our store. How do we go about it?

PS: Aware that we can build a custom checkout but that's not something we are doing right now. We want to modify the existing checkout which is available via Swell.

Thanks

**Comment by ericingram** [comment]
Which fields would you want to remove? We might consider an improvement to support natively if it makes sense

**Reply by virajsoni06** [reply]
![image](https://github.com/swellstores/community/assets/13236286/959bd198-ff01-4faf-968a-809ad8c3b5b9)
Almost all the fields in the above screenshot except name

Also another issue is the user is requested to fill in their details twice, once on the above screen and again on the billing form (image below). This is quite inefficient and probably discourages users to finish the order. We want the checkout experience as frictionless as possible. I imagine all these fields can be combined into a single form. Its especially important for us since we aren't selling physical products and only digital products and don't really need much info about the user except Email, Name & Country

![image](https://github.com/swellstores/community/assets/13236286/340d329f-669e-4dce-8a2e-76f1c948e02c)

**Reply by ericingram** [reply]
We are working on this now

**Reply by virajsoni06** [reply]
@ericingram any eta on when that would be available?

**Reply by ericingram** [reply]
cc @aaronlee777

**Reply by virajsoni06** [reply]
Hi @ericingram @aaronlee777 any update on this?

**Reply by ericingram** [reply]
We haven't been able to prioritize this but will discuss it more internally today

**Reply by aaronlee777** [reply, marked as answer]
Hi @virajsoni06! You should be able to turn off those first few fields you mentioned. To do so, navigate to Storefronts > Checkout in the admin dashboard. If you scroll down you'll see there is a section for form fields where you can toggle off company name, phone number and customer comments.

Right now we don't have plans to modify the billing address section of the checkout.
